assets/src/design-system/components/input/useNumericInput.js

```diff
@@ -32,6 +32,7 @@ export const useNumericInput = ({
   value,
 }) => {
   const inputRef = useRef(null);
+  const changeTracker = useRef(false);
```

Comment:
I don't think we need this extra ref since we already have a ref that handles if the value should be updated or not. It needed to be tweaked a little but I think this works: https://github.com/google/web-stories-wp/pull/6604/files#diff-be15e4e815ede5e372dcb412f677988697bb6f47359a6b97dea5d366c06dbd9eL36-R121

This doesn't depend on `isIndeterminate`. Regardless of if we keep `isIndeterminate` I think this is functionally equivalent 😄

Reply:
@samwhale Not sure what you mean exactly. For detecting if the value should be updated or not, only these are the relevant parts if that's what you mean:
<img width="593" alt="Screenshot 2021-03-05 at 09 14 13" src="https://user-images.githubusercontent.com/3294597/110080475-cb051980-7d8a-11eb-9786-dc8b1af281bc.png">

The `changeTracker` is purely for the `isIndeterminate`. True that this could also work without the `changeTracker` though -- originally I added the `changeTracker` since if the value changes back to what it was before then it has still changed so we can't just compare the old and the new value, however, if it changes back to exactly how it was, then we should again have `isIndeterminate` as `true` if it was originally.